default.nix

```diff
@@ -0,0 +1,93 @@
+{
+  python3Packages,
+  cmake,
+}: let
+  fqfa = with python3Packages;
```

Comment:
I don't know much about Nix, and I suspect this is Just How It Is Done In Nix, but is there any way to prevent/detect these versions getting out of step with the versions in setup.py?

Reply:
Sadly no, with the base nixpkgs only there's no mechanism available to read python requirements and solve the constraint problem (nixpkgs does not allow dynamic resolution of dependencies, i.e., import from derivation in nix lingo, for some good reasons). This is what's done in more complex solutions (like https://github.com/DavHau/mach-nix) but the dependencies here are not numerous so I think it's worth sticking to the far simpler nixpkgs solution and updating manually as needed.

Reply:
Yeah, probably.  I guess my real question is: will the Nix package build process break if they get out of step?
If so we can add a CI rule for it or something, thus reminding us to fix it.

Reply:
Yes it can break, CI is a good idea. Not sure what CI you're currently using for this project, but there's a github workflow that will build the flake that's easy to setup https://github.com/marketplace/actions/install-nix#usage-with-flakes. Would you like me to add that to this PR?

Reply:
I've disabled the install checks for tables, it not a good tradeoff to do this repeatedly in the CI and also isn't a great user experience for somebody running CountESS to have to wait ~20m for it to start. When tables 3.8.0 is in nixpkgs we can bump the flake and the problem will go away. Build down to ~5m: https://github.com/jbedo/CountESS/actions/runs/4257342035/jobs/7407356911